### ISSUE TITLE
Write up instructions for using with a notebook 

### ISSUE THREAD

**Issue by henridf** [henridf/apache-spark-node#22]
Beaker seems like the best candidate, given its support for javascript and its polyglot nature. 

My guess/hope is that this should be pretty simple...

**Comment by itayw** [comment]
Thanks for this project. 

May I suggest also looking into Apache Zeppelin?
https://zeppelin.incubator.apache.org/

**Comment by tobilg** [comment]
@itayw As far as I know Zeppelin already incorporate Spark. So what exactly do you expect?

**Comment by itayw** [comment]
Yep, they do have native support for Spark.

I'm still learning about this project and when, so this issue maybe out of
place. However, when I saw this ticket I thought it would be great to
somehow combine between Zeppelin and its capabilities and having the
freedom to use JavaScript instead of Java, Scala or python.

**Comment by henridf** [comment]
@itayw yes Zeppelin seems like another good candidate. I don't know if it already has javascript support though? Whichever one (Zeppelin, Beaker, ...) my hope is that this shouldn't require any dev effort, just putting the pieces together...

**Comment by itayw** [comment]
Here's a gist for building a Docker image that will allow you to run `beaker` with `apache-node-spark`.

https://gist.github.com/itayw/553effc4ee0cff5d305e

I've used beaker's image coupled with instructions taken from apache-node-spark Dockerfile. I managed to run snippets based on `test/smokeTest.js` without special issues.

I'm continuing to learn more about the integration and ease-of-use. Please let me know of any feedback.

**Comment by tobilg** [comment]
@itayw Thanks, looks good after a short first look. I currently update the standard Dockerfile concerning new Spark and Node.js versions. I'll open a PR for this today.

Going forward, I'd suggest to have two different Docker images, one for the standard REPL, and one for the Beaker notebook. @henridf could publish both on Docker Hub.
